src/InteractiveWindow/VisualStudio/VsInteractiveWindowCommandFilter.cs

```diff
@@ -269,6 +269,11 @@ private int PreEditorCommandFilterExec(ref Guid pguidCmdGroup, uint nCmdID, uint
             {
                 switch ((VSConstants.VSStd97CmdID)nCmdID)
                 {
+                    // TODO: Add support of rotating clipboard ring 
+                    // https://github.com/dotnet/roslyn/issues/5651
+                    case VSConstants.VSStd97CmdID.PasteNextTBXCBItem:
+                        return VSConstants.S_OK;
```

Comment:
I think there's a different HRESULT for not handling a command.  E_FAIL?   S_FALSE?